roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -77,6 +77,17 @@
     - '"fwaas-v1" not in enable_services'
     - '"fwaas-v2" not in enable_services'
 
+- name: enable q-lbaasv2 with octavia enabled for stable/mitaka branch
+  shell:
+    cmd: |
+      set -e
+      set -x
+      cat << EOF >> /tmp/dg-local.conf
+      enable_service q-lbaasv2
+      EOF
+    executable: /bin/bash
+  when: global_env.OS_BRANCH == 'stable/mitaka' and '"octavia" in enable_services'
```

Comment:
We have a code block to add options when `octavia` is enabled in line 204, I prefer to add some logic in there, something like:
```bash
if [[ "{{ global_env.OS_BRANCH }}" == "stable/mitaka" ]]
    enable_service q-lbaasv2
```